src/Form/Input.js

```diff
@@ -46,7 +60,19 @@ const StyledInput = createComponent({
 
     ${isFloating &&
       css`
-        padding-bottom: 0;
+        line-height: ${getOneSizeLarger(theme.fontSizes, size)}px;
```

Comment:
It seems to me that these utilities (`getOneSizeLarger`/`getOneSizeSmaller`) are a bit confusing and introducing unnecessary complexity. Have you considered using native css `calc()` to achieve a similar result here and elsewhere these utils are used?

```css
css`
  line-height: calc(${theme.fontSizes[size]}px * 1.2);
  padding-top: calc(${theme.fontSizes[size]}px * 1.2);
`
```

Reply:
That was my original solution, but since Jay had made very specific outlines to what the different forms of spacing/sizing needed to be, I created these utils in order to achieve that accurately instead of using `calc`. But, if Jay has no problem with it, I'm game.

Reply:
I'll switch to `calc` and have him review.

Reply:
The main place I see a problem is when these utils are changing font sizes.